test/server/cards/01_SOR/units/FrontlineShuttle.spec.ts

```diff
@@ -0,0 +1,87 @@
+describe('Frontline Shuttle', function() {
+    integration(function(contextRef) {
+        describe('Frontline Shuttle\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        spaceArena: ['frontline-shuttle'],
+                        groundArena: ['battlefield-marine', 'r2d2#ignoring-protocol', 'c3po#protocol-droid']
+                    },
+                    player2: {
+                        groundArena: ['bail-organa#rebel-councilor', 'sundari-peacekeeper'],
+                    }
+                });
+            });
+
+            it('should be able to activate action ability and attack with exhausted unit', function () {
+                const { context } = contextRef;
+
+                context.battlefieldMarine.exhausted = true;
```

Comment:
Can declare in this test setup like so:

```typescript
groundArena: [{ card: 'battlefield-marine', exhausted: true }, 'r2d2#ignoring-protocol', 'c3po#protocol-droid']
```

Reply:
@AMMayberry1 Nice, I updated this test and will keep the same pattern in future implementations.